docs/sources/pip.md

```diff
@@ -0,0 +1,11 @@
+# Pip
+
+The pip source uses `pip` CLI commands to enumerate dependencies and properties. It is expected that `pip` is available in the PATH before running `licensed`
+
+#### virtual_env_dir
+A `virtualenv` is assumed to be setup before running `licensed`. The `pip` command will be sourced from this directory.
```

Comment:
> A `virtualenv` is assumed to be setup before running `licensed`.

Can you move ☝️ into the top section as another requirement/expectation?

Reply:
Also can you make note that the `virtual_env_dir` setting is used to find the virtualenv,  that it's a relative path to the repository root, and that the setting is required for the source to work properly?